src/input_context/preset.rs

```diff
@@ -0,0 +1,173 @@
+use bevy::prelude::*;
+use serde::{Deserialize, Serialize};
+
+use crate::{Negate, SwizzleAxis};
+
+use super::bind::{BindConfigs, IntoBindConfigs};
+
+/// Maps 4 buttons as 2-dimentional input.
+///
+/// This is a convenience preset that uses [`SwizzleAxis`] and [`Negate`] to
+/// bind the buttons to X and Y axes.
+///
+/// # Examples
+///
+/// Map keyboard inputs into a 2D movement action
+///
+/// ```
+/// use bevy::prelude::*;
+/// use bevy_enhanced_input::prelude::*;
+///
+/// #[derive(Debug, Resource)]
+/// struct ControlSettings {
+///     forward: Vec<Input>,
+///     right: Vec<Input>,
+///     backward: Vec<Input>,
+///     left: Vec<Input>,
+/// }
```

Comment:
Let's make the example more common with a clarification:
```suggestion
// In games different input sources usually have different bindings.
// But also possible to `Input` to consolidate different input sources in one setting.
/// #[derive(Debug, Resource)]
/// struct KeyboardSettings {
///     forward: Vec<KeyCode>,
///     right: Vec<KeyCode>,
///     backward: Vec<KeyCode>,
///     left: Vec<KeyCode>,
/// }
```

Reply:
Changed and added comments